lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -38,9 +38,19 @@ def descriptions(version)
     end
 
     def all_questions
-      dataset.keys.inject({}){|versions, version| versions.merge(questions(version))}
+      dataset.keys.inject({}) { |versions, version| versions.merge(questions(version)) }
     end
 
+    def is_london?(postcode)
+      area(postcode).any? { |result| result[:slug] == "london" }
+    end
+
+    def area(postcode)
+      response = Services.imminence_api.areas_for_postcode(postcode)
+      (response&.results || {})
+    end
```

Comment:
Minor: I'd prefer it if we duplicated *exactly*  the code in [LandlordImmigrationCheckCalculator#areas_for_postcode][1] here to make it more obvious that this is ripe for refactoring. Ultimately I think we should extract some of this code into some kind of service adapter class which would wrap calls to `GdsApi::Imminence` in a friendlier interface.

[1]: https://github.com/alphagov/smart-answers/blob/0f9d51b3022f10a071b07a393a9802e701293d21/lib/smart_answer/calculators/landlord_immigration_check_calculator.rb#L21-L24